mappings/net/minecraft/datafixer/fix/BitStorageAlignFix.mapping

```diff
@@ -1,3 +1,47 @@
 CLASS net/minecraft/class_5196 net/minecraft/datafixer/fix/BitStorageAlignFix
+	FIELD field_29863 ELEMENT_BIT_SHIFT I
+	FIELD field_29864 UNUSED_CHUNK_WIDTH I
+	FIELD field_29865 UNUSED_CHUNK_DEPTH I
+	FIELD field_29866 MAX_BLOCK_STATE_ID I
+	FIELD field_29867 HEIGHT_VALUE_BITS I
+	FIELD field_29868 MAX_HEIGHT_VALUE I
 	METHOD <init> (Lcom/mojang/datafixers/schemas/Schema;)V
 		ARG 1 outputSchema
+	METHOD method_27288 resizePackedIntArray (II[J)[J
+		ARG 0 maxValue
+		ARG 1 elementBits
+		ARG 2 elements
+	METHOD method_27292 (Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;
+		ARG 5 chunk
+	METHOD method_27293 (Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;
+		ARG 4 level
+	METHOD method_27294 (Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;
+		ARG 2 levelSection
+	METHOD method_27295 (Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;
+		ARG 1 section
+	METHOD method_27296 (Ljava/util/List;)Ljava/lang/Integer;
+		ARG 0 pallete
+	METHOD method_27769 (Lcom/mojang/serialization/Dynamic;)Lcom/mojang/serialization/Dynamic;
+		ARG 0 levelDynamic
+	METHOD method_27772 fixBitStorageArray (Lcom/mojang/serialization/Dynamic;Lcom/mojang/serialization/Dynamic;II)Lcom/mojang/serialization/Dynamic;
+		ARG 0 sectionDynamic
+		ARG 1 statesDynamic
+		ARG 2 maxValue
+		ARG 3 elementBits
+	METHOD method_27773 (Lcom/mojang/serialization/Dynamic;Lcom/mojang/datafixers/util/Pair;)Lcom/mojang/datafixers/util/Pair;
+		ARG 1 heightmap
+	METHOD method_27774 fixLevel (Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/OpticFinder;Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;
+		ARG 0 levelSectionsFinder
+		ARG 1 sectionFinder
+		ARG 2 palletteFinder
```

Comment:
```suggestion
		ARG 2 paletteFinder
```

Reply:
ah yes, i make similar typos and can't spot them apparently!